tests/test_dirichlet_distributions.py

```diff
@@ -348,3 +348,48 @@ def test_geodesic_vectorization(self):
         result = geod(time).shape
         expected = (self.n_points, self.dim)
         self.assertAllClose(expected, result)
+
+    @geomstats.tests.np_only
+    def test_jac_christoffels(self):
+        """Test jacobian of Christoffel symbols.
+
+        Check vectorization.
+        """
+        base_point = self.dirichlet.random_point()
+        result = self.metric.jac_christoffels(base_point)
+        self.assertAllClose(
+            (self.dim, self.dim, self.dim, self.dim),
+            result.shape)
+
+        base_points = self.dirichlet.random_point(2)
+        result = self.metric.jac_christoffels(base_points)
+        expected = [
+            self.metric.jac_christoffels(base_points[0, :]),
+            self.metric.jac_christoffels(base_points[1, :])]
+        expected = gs.stack(expected, 0)
+        self.assertAllClose(expected, result)
+
+    @geomstats.tests.np_only
+    def test_jacobian_in_geodesic_bvp(self):
+        """Test Jacobian option in geodesic bvp.
+
+        Check that dist yields the same result with
+        and without.
+        """
+        point_a = self.dirichlet.random_point()
+        point_b = self.dirichlet.random_point()
+        result = self.dirichlet.metric.dist(point_a, point_b, jacobian=True)
+        expected = self.dirichlet.metric.dist(point_a, point_b)
+        self.assertAllClose(expected, result)
+
+    @geomstats.tests.np_only
+    def test_geodesic_bvp_timer(self):
```

Comment:
If you have an easy example on this: would it be possible to also write a test for `jac` of bvp function?

Reply:
Yes it would be nice to test this, with autograd for example, but I would need to make it a class method, right ? As it is, I don't see how I can test it since I can't access it. What I did for now, is I tested that the bvp solver yields the same result, with and without using the jacobian.